tictactoe/src/store.erl

```diff
@@ -0,0 +1,27 @@
+-module(store).
+-export([store_move/3,undo_store_move/2]).
+-include_lib("eunit/include/eunit.hrl").
+
+store_move(Location, Piece, Board) ->
+  H = lists:sublist(Board, Location - 1),
```

Comment:
Try to be more expressive with your variable names. I assume H is head and T is tail, but you could probably express what those represent better.